### ISSUE TITLE
Release 5.20.0 is broken

### ISSUE THREAD

**Issue by segevfiner** [googleapis/nodejs-storage#1931]
#### Environment details

  - OS: macOS 12.3.1
  - Node.js version: 16.15.0
  - npm version: 8.10.0
  - `@google-cloud/storage` version: 5.20.0

#### Steps to reproduce

  1. Download https://registry.npmjs.org/@google-cloud/storage/-/storage-5.20.0.tgz
  2. There is no JS code in the file...

```
Cannot find module '@google-cloud/storage' or its corresponding type declarations.
```

**Comment by mcarriere** [comment]
same issue here

**Comment by ddelgrosso1** [comment]
@segevfiner this should be resolved by `v5.20.1`. If you are still seeing issues with `5.20.1` please reopen this issue and I will investigate further.
